doc/examples/edges/plot_ridge_filter.py

```diff
@@ -0,0 +1,76 @@
+"""
+===============
+Ridge operators
+===============
+
```

Comment:
Please comment that each filters is best suited to detect a different shape of object

Reply:
Do you mean something general like "Different ridge filters may be suited for detecting different structures, depending, e.g., on contrast or noise level."? I can't say anything more specific, see https://github.com/scikit-image/scikit-image/pull/3515#pullrequestreview-169053886